core/storage/app_feedback_report/gae_models.py

```diff
@@ -43,26 +43,21 @@
 GITHUB_REPO_CHOICES = PLATFORM_CHOICES
 
 
-# TODO(#14419): Change naming style of Enum class from SCREAMING_SNAKE_CASE
-# to PascalCase and its values to UPPER_CASE. Because we want to be consistent
-# throughout the codebase according to the coding style guide.
-# https://github.com/oppia/oppia/wiki/Coding-style-guide
-
 # The model field names that can be filtered / sorted for when maintainers
 # triage feedback reports.
-class FILTER_FIELD_NAMES(enum.Enum): # pylint: disable=invalid-name
+class FilterFieldNames(enum.Enum):
     """Enum for the model field names that can be filtered"""
 
-    platform = 'platform' # pylint: disable=invalid-name
-    report_type = 'report_type' # pylint: disable=invalid-name
-    entry_point = 'entry_point' # pylint: disable=invalid-name
-    submitted_on = 'submitted_on' # pylint: disable=invalid-name
-    android_device_model = 'android_device_model' # pylint: disable=invalid-name
-    android_sdk_version = 'android_sdk_version' # pylint: disable=invalid-name
-    text_language_code = 'text_language_code' # pylint: disable=invalid-name
-    audio_language_code = 'audio_language_code' # pylint: disable=invalid-name
-    platform_version = 'platform_version' # pylint: disable=invalid-name
-    android_device_country_locale_code = 'android_device_country_locale_code' # pylint: disable=invalid-name
+    PLATFORM = 'PLATFORM'
+    REPORTTYPE = 'REPORTTYPE'
```

Comment:
I'm not sure this is quite how we want to handle enums. Shouldn't they be REPORT_TYPE, etc.? The current version is a bit unreadable.

Also should the values be changed? What if existing models already use the previous values?

/cc @vojtechjelinek to weigh in, too.

Reply:
Hello, Sounds good! I'll have it changed. Just to clarify, values should be kept the same as they are in my PR or as they were prior to my PR? Thanks for the feedback/assistance. I will have it done shortly.

Reply:
The same as they were prior to your PR.

Reply:
Hi, Just to clarify one last  time before I submit the corrections. Values like report type should be reverted entirely back to lowercase(so for example)`   REPORTTYPE = 'REPORTTYPE'` should be `   REPORT_TYPE = 'report_type' ? Also `FilterFieldNames` Should be `Filter_Field_Names` ?

Reply:
The first one is correct.

The second one is not. It should remain as FilterFieldNames. It's possible to distinguish the individual words there because of the capital letters; this is called "upper camel case".

Reply:
OK Great thanks again for the clarification. I will follow these guidelines.

Reply:
Alright it should be fixed. Let me know if there are any more changes that are required!